osc-server/src/main/java/org/osc/core/broker/job/TaskNode.java

```diff
@@ -123,7 +123,10 @@ synchronized void setState(TaskState state) {
         }
 
         if (this.taskRecord != null) {
-            persistState();
+            // Maybe bug on Pessimistic lock
```

Comment:
can you add more detailed comment? Maybe a TODO or reference to a issue filed in the OSC repo?